docs/Node Operators/configuration-variables.md

```diff
@@ -420,6 +435,102 @@ Setting this number higher will cause Chainlink to select higher gas prices.
 
 Setting it lower will tend to set lower gas prices.
 
+# Debugging
+
+## Logging
+
+### JSON_CONSOLE
+
+- Default: `"false"`
+
+Set to true to enable JSON logging. Otherwise will log in a human-friendly console format.
+
+### LOG_LEVEL
+
+- Default: `"info"`
+
+The `LOG_LEVEL` environment variable determines both what is printed on the screen and what is written to the logfile, located at `$ROOT/log.jsonl`.
+
+The available options are:
+- "debug"
+- "info"
+- "warn"
+- "error"
+- "panic"
+
+### LOG_SQL
+
+- Default: `"false"`
+
+Tells Chainlink to log all SQL statements made using the default logger.
+
+### LOG_SQL_MIGRATIONS
+
+- Default: `"true"`
+
+Tells Chainlink to log all SQL migrations made using the default logger.
+
+### LOG_TO_DISK
+
+- Default: `"true"`
```

Comment:
This was switched off recently. https://github.com/smartcontractkit/chainlink/commit/639b65d9f6621ffb12286968520e3aa29d5c50c5
```suggestion
- Default: `"false"`
```

Reply:
Nice catch